### DISCUSSION TITLE
Maximum cell length

### DISCUSSION THREAD

**Discussion by TecLinkAutomacao** [doonfrs/pluto_grid_plus#75] [General]
How can I define a maximum number of characters in a cell? for example, I have a column called code and I want a maximum of 5 characters to be entered in it, example 12345, the sixth character cannot be entered.

**Comment by stan-at-work** [comment]
> How can I define a maximum number of characters in a cell? for example, I have a column called code and I want a maximum of 5 characters to be entered in it, example 12345, the sixth character cannot be entered.
You can use a custom renderer and put your own inputform on it.
